cypress/features/regression/flatTabel.feature

```diff
@@ -0,0 +1,88 @@
+Feature: FlatTable component
+  I want to check Flat Table component properties
+
+  Background: Open FlatTable component default page
+    Given I open "Flattable" component page
+
+  @positive
+  Scenario: FlatTable cell has row header
+    When I check hasRowHeader checkbox
+    Then FlatTable has row header
+
+  @positive
+  Scenario: FlatTable has sticky header
+    When I check hasStickyHeader checkbox
+    Then FlatTable has sticky header
+
+  @positive
+  Scenario: Check that Flat Table has nine rows
+    # commented because of BDD default scenario Given - When - Then
+    # When I open "Flattable" component page
+    Then FlatTable has nine row
+
+  @positive
+  Scenario: Check that Flat Table has seven columns
+    # commented because of BDD default scenario Given - When - Then
+    # When I open "Flattable" component page
+    Then FlatTable has nine row
+
+  @positive
+  Scenario Outline: Check value in header cells
+  # commented because of BDD default scenario Given - When - Then
+  When I open "Flattable" component page
```

Comment:
`When` step need to be commented? Or the `# commented because of BDD default scenario Given - When - Then` is unnecessary

Reply:
I commented it